my-app/src/AccountDetails.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import PaymentHistoryTable from './PaymentHistoryTable';
+
+interface AccountDetailsProps {
+  id: string;
+}
+
+interface AccountInfo {
+  name: string;
+  socialSecurityNumber: string;
+  dateOfBirth: string;
+  address: string;
+  creditScore: number;
+  email: string;
+  phoneNumber: string;
+  cardNumber: string;
+}
+
+interface AccountInfo {
+  id: string;
+  name: string;
+  socialSecurityNumber: string;
```

Comment:
This is probably a token, which you would need to detokenize. If it's not I would maybe look at the BE implementation and redo it, since this is important PII information that could be vulnerable.